app/client/templates/pages/homepage.html

```diff
@@ -1,5 +1,7 @@
 <template name="homepage">
   <div class="container">
-    <h2>Hello Cruel World</h2>
+    <p>Bienvenido al proceso de registro para participar en la XVII Bienal de Fotografía organizada por el Centro de la Imagen.</p>
+    <a href="/registrarse" class="waves-effect waves-light btn">Comenzar el registro</a>
+    <a href="/registro" class="waves-effect waves-light btn">Reanudar tu registro</a>
```

Comment:
Shouldnt this be conditional to `is logged in` and either take you to login or /registro? Even tho /resistro will redirect that seems less tidy.